DopplerCustomDomain.Test/DnsValidation/SystemDnsResolutionValidatorTest.cs

```diff
@@ -38,7 +38,6 @@ public abstract class DnsResolutionValidatorTest<Tsut>
         [InlineData("relaytrk.dopplerrelay.org")] // CNAME => trk.relaytrk.com
         [InlineData("dopplerpages.com")] // A
         [InlineData("www.dopplerpages.com")] // CNAME => dopplerpages.com
-        [InlineData("r198.ddns.net")] // CNAME => www.dopplerpages.com
```

Comment:
@andresmoschini I see that you made this test, and I assume that this DNS was used for testing service, now does not exist, and I suppose is safe to remove it, but I want your opinion. Thanks

Reply:
For the moment, I comment the line.